index.js

```diff
@@ -1,32 +1,65 @@
 'use strict';
 
 import {Platform, NativeModules} from "react-native";
-const DocumentPicker = NativeModules.RNDocumentPicker;
+const DocumentPicker = NativeModules.RNDocumentPicker || NativeModules.DocumentPickerModule;
```

Comment:
Naming your Windows module the same as the Android and iOS modules would be preferable to using a different name.

Reply:
Fixes: https://github.com/Elyx0/react-native-document-picker/pull/80/commits/9f2ae818b44bba8652caa570a5b48fe6cedae991